hal/targets/hal/TARGET_Freescale/TARGET_KSDK2_MCUS/api/objects.h

```diff
@@ -61,6 +61,10 @@ struct dac_s {
     DACName dac;
 };
 
+struct rng_s {
```

Comment:
I understand why mbedtls has a `void *` as a first parameter. Indeed, the API should even work when the value of the parameter is `NULL`. However, I am not entirely sure why this API has a first parameter which is a completely empty structure. Perhaps, at least there should be a way to tell whether it has been initialised or something like this?